python/tests/__init__.py

```diff
@@ -113,6 +113,22 @@ def _levelorder_nodes(self, u, l, level):
         for c in self.children(u):
             self._levelorder_nodes(c, l, level + 1)
 
+    def _minlex_postorder_nodes(self, u, l):
+        l.extend(self._minlex_postorder_nodes_helper(u)[1])
+
+    def _minlex_postorder_nodes_helper(self, u):
+        children = self.children(u)
+        if children:
+            children_return = [self._minlex_postorder_nodes_helper(c) for c in children]
+            children_return.sort()
+            postorder = []
+            for element in children_return:
```

Comment:
``element`` isn't very helpful here, how about 
```python
nodes = []
for _, child_nodes in sorted_children:
    nodes.extend(child_nodes)
nodes.append(u)
```

Reply:
I've opted for some longer names instead. Hopefully with the added docstring things are clear.